sessionctx/variable/sysvar.go

```diff
@@ -1160,7 +1160,7 @@ var defaultSysVars = []*SysVar{
 		s.EnableTablePartition = val
 		return nil
 	}},
-	{Scope: ScopeGlobal | ScopeSession, Name: TiDBEnableListTablePartition, Value: Off, Type: TypeBool, SetSession: func(s *SessionVars, val string) error {
+	{Scope: ScopeGlobal | ScopeSession, Name: TiDBEnableListTablePartition, Value: On, Type: TypeBool, SetSession: func(s *SessionVars, val string) error {
```

Comment:
This will only change the value for new installs, since the original value is saved on bootstrap. Is this okay?

Since this is a feature flag, I would actually suggest that you can instead remove the variable (see `sessionctx/variable/removed.go`), but I will leave that up to you :-)

Reply:
We would like to backport it to v6.1 and then remove it in v6.2, as just discussed.

Reply:
> This will only change the value for new installs, since the original value is saved on bootstrap. Is this okay?

IMHO it's OK